generated/nidcpower/session.py

```diff
@@ -2368,9 +2368,63 @@ def configure_aperture_time(self, aperture_time, units=enums.ApertureTimeUnits.S
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
-    def fetch_multiple(self, count, timeout=datetime.timedelta(seconds=1.0)):
+    def fetch_multiple(self, count=None, timeout=datetime.timedelta(seconds=1.0)):
```

Comment:
I've been thinking about the default `count=None`. I don't think it's the right thing to do.

The most common use case would be for the client to:

* Configure N measurements
* Initiate
* Fetch 

...with the intent of fetching N measurements. Given the default of "whatever is available", most of the time client will get 0 or no measurements.

I'm leaning towards no default for `count`.

Reply:
I agree. Default value for count removed.